MoreLinq/ScanBy.cs

```diff
@@ -40,14 +40,17 @@ static partial class MoreEnumerable
         /// <returns>
         /// A sequence of keys paired with intermediate accumulator states.
         /// </returns>
-
+        /// <exception cref="ArgumentNullException"><paramref name="source"/> is null</exception>
```

Comment:
Since it's the same exception listed in each case, you just want one `<exception>` block instead:

```c#
/// <exception cref="ArgumentNullException">
/// <paramref name="source"/>, <paramref name="keySelector"/>,
/// <paramref name="seedSelector"/> or <paramref name="accumulator"/> is
/// <see langword="keyword"/>.
/// </exception>
```

---
PS [See also how it's done for `Union`](https://github.com/dotnet/runtime/blob/v7.0.0/src/libraries/System.Linq/src/System/Linq/Union.cs#L52).